CHANGELOG.md

```diff
@@ -1,14 +1,36 @@
 **New Features and Improvements**
-* Add direct aiming from Crispy Doom. More accurate vertical aiming (thanks to @ceski-1).
-* Optimizations for plane rendering, taken from Eternity Engine (thanks to @JNechaevsky).
-* Add `-dedicated`, `-uncapped` and `-nouncapped` command line parmeters (thanks to @loopfz).
-* Rename `-nodehlump` to `-nodeh` for consistency with other ports.
-* Use sector lightlevel for sprites in Boom and Vanilla complevels.
+* 3D audio support (@ceski-1)
+  - HRTF mode ("Headphones mode" in the General menu).
```

Comment:
Small edit:
```
- Stereo and up to 7.1 surround sound.
- HRTF support ("Headphones mode" in the General menu).
```

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1208/commits/0a56da97df9c2860260b99c273c5b1bf6e2c47be